Tests/sbgTests/ApplicationTests.swift

```diff
@@ -205,16 +206,25 @@ class ApplicationTests: QuickSpec {
                 }
 
                 context("invokes project manipulator") {
+                    // (groupPath: String, fileName: String, xcodeprojFile: String, target targetName: String){
```

Comment:
What's the point of this comment?

Reply:
I used it when I refactored the tests, now it's obsolete, I will remove it.